modules/ossm-security-auth-policy.adoc

```diff
@@ -0,0 +1,130 @@
+
+* The `selector` field specifies the target of the policy.
+* The `action` field specifies whether to allow or deny the request.
+* The `rules` specify when to trigger the action.
+** The `from` field in the rules specifies the sources of the request.
+** The `to` field in the rules specifies the operations of the request.
+** The `when` field specifies the conditions needed to apply the rule.
+
+== Configure intra-project communication
+
+You can use authorization policies to configure your control plane to allow or deny the traffic communicating with your mesh or services in your mesh. 
+
+=== Restrict access to services outside a namespace
+
+You can deny requests from any source that is not in the `bookinfo` namespace with the following `AuthorizationPolicy` custom resource example.
+
+[source,yaml]
+----
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+ name: httpbin-deny
+ namespace: bookinfo
+spec:
+ selector:
+   matchLabels:
+     app: httpbin
+     version: v1
+ action: DENY
+ rules:
+ - from:
+   - source:
+       notNamespaces: ["bookinfo"]
+----
+
+=== Creating allow-all and default deny-all authorization policies
+
+The following example shows an allow-all authorization policy that allows full access to all workloads in the default namespace.
+
+[source,yaml]
+----
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: allow-all
+  namespace: bookinfo
+spec:
+  action: ALLOW
+  rules:
+  - {}
+----
+
+The following example shows a policy that doesn’t allow any access to all workloads in the `bookinfo` namespace.
+
+[source,yaml]
+----
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: deny-all
+  namespace: bookinfo
+spec:
+  {}
+----
+
+== Create a role for the ingress gateway
+
+== Set restricted access with JWT
+
+[source,yaml]
+----
+apiVersion: "security.istio.io/v1beta1"
+kind: "RequestAuthentication"
+metadata:
+  name: "jwt-example"
+  namespace: bookinfo
+spec:
+  selector:
+    matchLabels:
+      app: httpbin
+  jwtRules:
+  - issuer: "testing@secure.istio.io"
+    jwksUri: "https://raw.githubusercontent.com/istio/istio/release-1.8/security/tools/jwt/samples/jwks.json"
+----
+
+=== Authorizing based on identity
+
+You can further restrict who can see workflows. The following example shows how to allow only authenticated users to access a workflow. 
+
+[source,yaml]
+----
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+ name: httpbin
+ namespace: bookinfo
+spec:
+ selector:
+   matchLabels:
+     app: httpbin
+     version: v1
+ action: ALLOW
+ rules:
+ - from:
+   - source:
+       principals: ["*"]
+   to:
+   - operation:
+       methods: ["GET", "POST"]
+----
+
+Run the following command once you have written your custom resource to add your custom resource to your namespace. The namespace must match your `metadata.namespace` field.
```

Comment:
I think this part can be left out- and if not, we should avoid using the term "custom resource" - it might be confusing because it can also mean the Custom Resources we introduce with OSSM (e.g. the ServiceMeshControlPlane is a Custom Resource)